workshop/content/40-dotnet/50-table-viewer/400-expose-table.md

```diff
@@ -62,7 +62,7 @@ namespace CdkWorkshop
 
 ## Now we can access the table from our stack
 
-Go back to `CdkWorkshop.cs` and assign the `Table` property of the table viewer:
```

Comment:
CdkWorkshop is a directory not a file. This seems to be talking about `CdkWorkshopStack.cs`